oaknational/Oak-Web-Application
# PUPIL-117 feat(pupil-quiz): connect pupil quiz to data

## Description

connect pupils quiz to data for multiple choice questions

- List of changes

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2120--oak-web-application.netlify.thenational.academypupils/programmes/physics-secondary-ks4-foundation-aqa/units/measuring-waves/lessons/transverse-waves
2. Click on the quiz to try out quiz
3. You should see multiple quiz question quiz

## Checklist
